lib/compress/huf_compress.c

```diff
@@ -732,7 +751,10 @@ static size_t HUF_compressCTable_internal(
 typedef struct {
     unsigned count[HUF_SYMBOLVALUE_MAX + 1];
     HUF_CElt CTable[HUF_SYMBOLVALUE_MAX + 1];
-    HUF_buildCTable_wksp_tables buildCTable_wksp;
+    union {
```

Comment:
unfortunately, nameless `union` is not a `C90` feature.

Reply:
Bummer, I'll name it then